packages/main/src/themes/sap_fiori_3/BusyIndicator-parameters.css

```diff
@@ -0,0 +1,5 @@
+@import "../base/BusyIndicator-parameters.css";
+
+:root {
+    --ui5-busyindicator-color: var(--sapUiContentIconColor);	
```

Comment:
As in hcb and fiori_3 the color is the same:
```--ui5-busyindicator-color: var(--sapUiContentIconColor);```

You can add this to the base/BusyIndicator-parameters.css file



```
:root {
--ui5-busyindicator-color: var(--sapUiContentIconColor);	
}
```
Then, hcb and fiori_3 parameter css files would need just an import of the base parameter css. The belize parameter css is fine.